.github/container/Dockerfile.t5x.arm64

```diff
@@ -0,0 +1,226 @@
+# syntax=docker/dockerfile:1-labs
+# Example command to build manually:
+#   docker buildx build -f Dockerfile.t5x.arm64 --tag t5x --build-arg BASE_IMAGE=ghcr.io/nvidia/jax:mealkit-2023-12-12 .
+
+ARG BASE_IMAGE=ghcr.io/nvidia/jax:mealkit
+ARG REPO_T5X=https://github.com/google-research/t5x.git
+ARG REF_T5X=main
+ARG SRC_PATH_T5X=/opt/paxml
+
+###############################################################################
+## build array_record and grain which do not have working arm64 pip wheels
+###############################################################################
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} as wheel-builder
```

Comment:
Can we move the array_record build stage forward, before the grain build stage? I feel that gives better logical connection, as the two stages are directly related. On the contrary, the grain builder is internalized from another dockerfile and does not inherit anything from `wheel-builder`.

Reply:
It's the other way around -- `grain-builder` can at least reuse the base image, whereas the `array_record` really requires on a completely different base image.

I now put the `array_record-builder` stage first and combined `wheel-builder` and `grain-builder` into one stage.